### PR TITLE
Lumberjack native support fixes #1732

### LINKED ISSUE #1732: Lumberjack native support

### PR DESCRIPTION
**Pull request by zbendhiba**
[ ] An issue should be filed for the change unless this is a trivial change (fixing a typo or similar). One issue should ideally be fixed by not more than one commit and the other way round, each commit should fix just one issue, without pulling in other changes.
[ ] Each commit in the pull request should have a meaningful and properly spelled subject line and body. Copying the title of the associated issue is typically enough. Please include the issue number in the commit message prefixed by #.
[ ] The pull request description should explain what the pull request does, how, and why. If the info is available in the associated issue or some other external document, a link is enough.
[ ] Phrases like Fix #<issueNumber> or Fixes #<issueNumber> will auto-close the named issue upon merging the pull request. Using them is typically a good idea.
[ ] Please run mvn process-resources -Pformat (and amend the changes if necessary) before sending the pull request.
[ ] Contributor guide is your good friend: https://camel.apache.org/camel-quarkus/latest/contributor-guide.html

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Comment by ppalaga** [comment]
Could we perhaps add some (e.g. `.bin` ) extension to the `window*` files so that we can easily exclude them from license checking around here https://github.com/apache/camel-quarkus/blob/master/pom.xml#L406-L416 ?

**Comment by zbendhiba** [comment]
@ppalaga I've changed to .bin and added  <exclude>**/*.bin</exclude> in pom parent

File: integration-tests/lumberjack/pom.xml [hunk 1]
```diff
@@ -51,6 +51,14 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-bean</artifactId>
```
Comment (jamesnetherton) [inline integration-tests/lumberjack/pom.xml:56]: Whenever you add camel extension dependencies into the itest pom, please run `mvn process-resources -Pformat -N` from the project root to regenerate the set of imported pom dependencies (E.g the stuff under the `The following dependencies guarantee that this module is built after them` comment).

Comment (zbendhiba) [reply integration-tests/lumberjack/pom.xml:56]: done !!
thanks !!

**Pull request merged**